app/presenters/working_group_presenter.rb

```diff
@@ -33,4 +33,13 @@ def extra_headings
     extra_headings << { id: "contact-details", text: "Contact details" } if email.present?
     extra_headings
   end
+
+  def first_item
+    return if body.nil?
+    if parsed_body.css("h2").empty?
+      parsed_body.css("div").first.first_element_child
```

Comment:
Similar to my question/comment above - is there a chance `parsed_body.css('div')` is ever empty or has no `first_element_child`?

Reply:
My thinking here is that there would always be at least a govspeak div in the body, but as above we should probably guard against it just in case.